scripts/tests_analyzer/compare_coderabbit_decisions.py

```
@@ -0,0 +1,689 @@
+    # Get issue comments (regular PR comments)
+    while True:
+        url = f"{GITHUB_API_BASE}/repos/{repo}/issues/{pr_number}/comments?per_page={per_page}&page={page}"
+        try:
+            data = github_request(url=url, token=token)
+            if not data or not isinstance(data, list):
+                break
+            comments.extend(data)
+            if len(data) < per_page:
+                break
+            page += 1
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)